src/SystemLoader.cc

```diff
@@ -126,6 +128,7 @@ class ignition::gazebo::SystemLoaderPrivate
 SystemLoader::SystemLoader()
   : dataPtr(new SystemLoaderPrivate())
 {
+  this->dataPtr->loader.SetFlags(RTLD_LAZY | RTLD_GLOBAL);
```

Comment:
This line here (I think) is going to generate some conflcts. We are openning all system libraries with this flag which will generate the same problem than my PRs.

We should change the flag inside the system plugin code. In particular `sensor` which is the system plugin that load the rendering sensors (camera, rgbd, etc).

Reply:
How are the system plugins failing for you? I tried running `sensors_demo` and `rolling_shapes` and they seem to be working for me. Be sure to recompile all libraries from plugin to gazebo.

Reply:
Removed